src/vm/jitinterface.cpp

```diff
@@ -11623,19 +11623,18 @@ void CEEJitInfo::allocMem (
 
     S_SIZE_T totalSize = S_SIZE_T(codeSize);
 
+    size_t roDataAlignment = ((flag & CORJIT_ALLOCMEM_FLG_RODATA_16BYTE_ALIGN)!= 0) 
+                             ? 16 : (roDataSize >= 8) ? 8 : sizeof(void*);
```

Comment:
It may be better to write this as series of if statements so that it looks (almost) the same as the code in zapinfo.cpp.